easybuild/easyconfigs/y/Yambo/Yambo-4.1.3-CrayIntel-2016.11.eb

```diff
@@ -0,0 +1,46 @@
+# contributed by Luca Marsella (CSCS)
+easyblock = 'MakeCp'
+
+name = 'Yambo'
+version = '4.1.3'
+
+homepage = 'http://www.yambo-code.org'
+description = """Yambo is a FORTRAN/C code for Many-Body calculations in solid state and molecular physics.
+ Yambo relies on the Kohn-Sham wavefunctions generated by two DFT public codes: abinit and PWscf."""
+
+toolchain = {'name': 'CrayIntel', 'version': '2016.11'}
+toolchainopts = {'usempi': True, 'openmp': True}
+
+sources = ['%s.tar.gz'%version]
+source_urls = ['https://github.com/yambo-code/yambo/archive/']
+
+dependencies = [
+    ('cray-hdf5/1.10.0', EXTERNAL_MODULE),
+    ('cray-netcdf/4.4.1', EXTERNAL_MODULE),
+    ('fftw/3.3.4.10', EXTERNAL_MODULE),
+    ('libxc', '3.0.0')
+]
+
+with_configure = 'True'
+
+preconfigopts = ' sed -i "s/\*ftn\* |//" configure && '
+
+configopts = ' CC=cc FC=ftn MPICC=cc CPPFLAGS="" FCFLAGS="-nofor_main -qopenmp" --enable-iotk --enable-open-mp '
+configopts += ' --with-hdf5-includedir="$HDF5_DIR/include" --with-hdf5-libdir="$HDF5_DIR/lib" '
```

Comment:
`$HDF5_DIR` is defined as the mapping of the external hdf5 module from Cray.
Why not use `$EBROOTHDF5`, since it is also defined when we do the map?
This would be a bit more generic.
What do you think?